### PR TITLE
DTSERWFOUR-24 Create Transfer - Display correct formatting for supported currencies

### PR DESCRIPTION
**Pull request by shanavas123**

### CODE REVIEW

File: transferui/src/main/java/com/hyperwallet/android/ui/transfer/CurrencyDetails.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    private String baseunit;
+    private Integer denominationamount;
+    private Integer decimals;
+    private Integer hiddenDecimals;
+    private String symbol;
+    private Integer exchangeable;
+    private Integer governmentissued;
+    private Integer groupingUsed;
+    private Integer fxTransactionVisible;
+    private String displayedAs;
```
Comment (vwagh-hw) [inline transferui/src/main/java/com/hyperwallet/android/ui/transfer/CurrencyDetails.java:19]: follow java naming conventions, for ex.
`isocurrencyCode` should be `isoCurrencyCode`
`denominationamount` should be `denominationAmount`
etc..

File: transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModel.java [hunk 2]
```diff
@@ -71,6 +72,9 @@
     private static final String DESTINATION_AMOUNT_INPUT_FIELD = "destinationAmount";
     private static final String DESTINATION_TOKEN_INPUT_FIELD = "destinationToken";
     private static final String PRIVATE_TOKEN_PREFIX = "trm-";
+    public static final String CURRENCY_DOT_SEPARATOR = ".";
+    public static final String REGEX_ONLY_NUMBER_AND_DECIMAL = "[^0-9.]";
```
Comment (vwagh-hw) [inline transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModel.java:76]: I don't see any usages for `REGEX_ONLY_NUMBER_AND_DECIMAL`

Comment (shanavas123) [reply transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModel.java:76]: removed

File: transferui/src/main/java/com/hyperwallet/android/ui/transfer/CurrencyDetails.java (hunk 1 above)
Comment (shanavas123) [reply transferui/src/main/java/com/hyperwallet/android/ui/transfer/CurrencyDetails.java:19]: fixed

**Review by rajkumar-balakrishnasamy — APPROVED** [review]

**Review by pdamodarannair — APPROVED** [review]

**Review by rajkumar-balakrishnasamy — APPROVED** [review]

**Review by vwagh-hw — APPROVED** [review]

**Pull request merged**